web/src/pages/barcoding.js

```diff
@@ -1,9 +1,119 @@
-import React from 'react'
+import React from 'react';
+import tw from 'twin.macro';
+import subway from '../images/barcoding/subway-takeover.png';
+import mask from '../images/barcoding/Mask-Group.png';
+import welcome from '../images/barcoding/Welcome-Box-2.png';
+import styled from 'styled-components';
+import billboard from '../images/barcoding/Billboard-mockup-1.png';
+import masktwo from '../images/barcoding/Mask-Group(1).png';
+import office from '../images/barcoding/Office.png';
+import signage from '../images/barcoding/Signage.png';
+import pattern from '../images/barcoding/Pattern.png';
+import lockup from '../images/barcoding/Barcoding-LockUp.png';
+import video from '../images/barcoding/Video-pic.png';
+import postcard1 from '../images/barcoding/Postcard-BG1.png';
+import postcard2 from '../images/barcoding/Postcard-BG2.png';
+import document from '../images/barcoding/B-document.png';
+import exploration from '../images/barcoding/B-Exploration.jpg';
+import tape from '../images/barcoding/tape-pattern.png';
+
+const HeroStyle = styled.div`
+  width: 100vw;
+  ${tw`w-screen`};
+  top: -160px;
+  margin-bottom: 100px;
+`;
+
+const ImgFlex = styled.div`
+  ${tw`grid grid-cols-2 gap-8 mx-24 mb-24`};
+`;
+
+const BarGrid = styled.div`
+  ${tw`grid grid-cols-2 gap-8 mx-24 mb-24`}
+  .item-b {
+    grid-column: 2 / 3;
+    grid-row: 1 / span 2;
+  }
+  .signage {
+    grid-row: 3 / span 1;
+    grid-column: 1 / 3;
+  }
+  .pattern,
+  .lockup {
+    grid-column: 1 / 3;
+  }
+  .video {
+    grid-column: 1 / 3;
+    grid-row: 5 / span 2;
+  }
+`;
+
+const DetailStyle = styled.div`
+${tw`flex flex-auto justify-between mb-24 mx-24`};
+  .portfolio-name {
+    font-weight: bold;
+    font-size: 50px;
+    line-height: 75px;
+    color: #2e444f;
+  }
+  p {
+    font-size: 22px;
+    width: 820px;
+    margin-top: 12px;
+    font-weight: 300;
+  }
+`;
+
+const BottomStyle = styled.div`
+  ${tw`flex flex-auto justify-between mx-24`};
+  .tape,
+  .document,
+  .mark {
+    height: 350px;
+  }
+`;
 
 export default function BarcodingPage() {
-    return (
-        <div>
-            <h1>this is the Barcoding Page</h1>
-        </div>
-    )
+  return (
+    <>
+      <HeroStyle>
+        <img src={subway} alt="subway" />
```

Comment:
Importing the images and accessing them like this is make the initial page load really slow. Can we try querying for them with a page query and passing them as props? If you want me to walk through one of these queries with you let me know.

Reply:
@asasmith done!